nexus/pkg/artifacts/saver.go

```diff
@@ -20,7 +20,7 @@ type ArtifactSaver struct {
 	// Resources.
 	Ctx           context.Context
 	GraphqlClient graphql.Client
-	UploadManager *uploader.UploadManager
+	UploadManager *filetransfer.FileTransferManager
```

Comment:
Nit: let's rename this as well
```suggestion
	FileTransferManager *filetransfer.FileTransferManager
```